hack/externalnode/install-vm.sh

```diff
@@ -120,10 +129,38 @@ Restart=on-failure
 WantedBy=multi-user.target
 EOF
     systemctl daemon-reload
-    systemctl enable antrea-agent
-    echo "Starting antrea-agent service"
-    systemctl start antrea-agent
-    systemctl status antrea-agent
+    systemctl enable "$ANTREA_AGENT"
+    echo "Starting ${ANTREA_AGENT} service"
+    systemctl start "$ANTREA_AGENT"
+    systemctl status "$ANTREA_AGENT"
+}
+
+start_antrea_monitor_service() {
```

Comment:
Feels the cleanup is more like a crontab job after reboot, other than a monitor service. The same for Windows.

Reply:
On Linux, crontab is the option, and on Windows the powershell commands `Register-ScheduledTask` and `New-JobTrigger` may help.

Reply:
yes, currently its only used for cleanup and does not justify the antrea-monitor name. (May be in future if there is need to add more features)
Do you prefer to have any other name for the antrea-monitor service?

Reply:
Firstly, I don't think using a system Service for such one-time task which is expected to be triggered after startup is a good option. Secondly, I prefer something like ovs-cleanup personally.

Reply:
> 

Crontab is an option which I explored a bit. It would require configuring the crontab to a specific user.
Since we already have a OS specific dependency with Service, I did not want to add additional OS specific dependencies.

Also on some version of Windows 10 based OSes, I have observed that Scheduled tasks configured to run at system startup would only run when a user logs in to the system. I will share more details about it offline.

Reply:
> Firstly, I don't think using a system Service for such one-time task which is expected to be triggered after startup is a good option. Secondly, I prefer something like ovs-cleanup personally.

Just renaming to ovs-cleanup may be confusing, I prefer renaming it to antrea-cleanup service.